--- awscli-1.25.97/setup.cfg.orig	2023-10-22 20:27:04.655307574 +0200
+++ awscli-1.25.97/setup.cfg	2023-10-22 20:36:03.419055507 +0200
@@ -4,10 +4,10 @@ universal = 0
 [metadata]
 requires_dist = 
 	botocore==1.27.96
-	docutils>=0.10,<0.17
+	docutils>=0.10
 	s3transfer>=0.6.0,<0.7.0
 	PyYAML>=3.10,<5.5
-	colorama>=0.2.5,<0.4.5
+	colorama>=0.2.5
 	rsa>=3.1.2,<4.8
 
 [check-manifest]
--- awscli-1.25.97/setup.py.orig	2023-10-22 20:27:04.658640889 +0200
+++ awscli-1.25.97/setup.py	2023-10-22 20:36:33.962223374 +0200
@@ -25,10 +25,10 @@ def find_version(*file_paths):
 
 install_requires = [
     'botocore==1.27.96',
-    'docutils>=0.10,<0.17',
+    'docutils>=0.10',
     's3transfer>=0.6.0,<0.7.0',
     'PyYAML>=3.10,<5.5',
-    'colorama>=0.2.5,<0.4.5',
+    'colorama>=0.2.5',
     'rsa>=3.1.2,<4.8',
 ]
 
